Common/src/main/resources/assets/crafttweaker/lang/it_it.json

```diff
@@ -0,0 +1,103 @@
+  "crafttweaker.command.description.docs" : "Apre un link alla documentazione",
+  "crafttweaker.command.description.dump" : "Salva un elenco dei sottocomandi disponibili per il comando /dump",
+  "crafttweaker.command.description.dump.brackets" : "Salva le espressioni tra parentesi uncinate disponibili nella cartella /ct_dumps",
+  "crafttweaker.command.description.dump.info" : "Restituisce un elenco di tutte le espressioni tra parentesi uncinate di tipo '%s' disponibili",
+  "crafttweaker.command.description.dump.loot_modifiers" : "Restituisce i nomi di tutti i modificatori di oggetti registrati",
+  "crafttweaker.command.description.dump.loot_tables" : "Restituisce i nomi di tutte le tabelle di oggetti registrate",
+  "crafttweaker.command.description.dump.biomes" : "Restituisce i nomi di tutti i biomi registrati",
+  "crafttweaker.command.description.dump.recipes" : "Restituisce i nomi di tutte le ricette registrate",
+  "crafttweaker.command.description.dump.tag.contents" : "Restituisce i nomi di tutti i tag registrati",
+  "crafttweaker.command.description.dump.villager.trades" : "Restituisce informazioni su tutti gli scambi dei villici",
+  "crafttweaker.command.description.dump.wandering.trades" : "Restituisce informazioni su tutti gli scambi dei mercanti ambulanti",
+  "crafttweaker.command.description.dump.fake_players" : "Restituisce informazioni su tutti i giocatori simulati",
+  "crafttweaker.command.description.dump.tool_tiers" : "Restituisce informazioni su tutti i livelli degli attrezzi",
+  "crafttweaker.command.description.examples" : "Crea script di esempio in base alle mod installate e apre la cartella degli esempi",
+  "crafttweaker.command.description.give" : "Dà un oggetto a un giocatore usando la sintassi del gestore di espressioni tra parentesi uncinate. Puoi anche applicare dei tag chiamando la funzione .withTag() alla fine.",
+  "crafttweaker.command.description.hand" : "Restituisce il nome e i tag (se esistono) dell'oggetto nella tua mano",
+  "crafttweaker.command.description.hand.attributes" : "Restituisce i modificatori di attributi dell'oggetto nella tua mano",
+  "crafttweaker.command.description.hand.data" : "Restituisce i dati formattati dell'oggetto nella tua mano",
+  "crafttweaker.command.description.hand.registryname" : "Restituisce il nome del registro dell'oggetto nella tua mano",
+  "crafttweaker.command.description.hand.tags" : "Restituisce i tag dell'oggetto nella tua mano",
+  "crafttweaker.command.description.hand.vanilla" : "Restituisce il nome e i tag (se esistono) dell'oggetto nella tua mano nel formato vanilla",
+  "crafttweaker.command.description.help" : "Trascrive la lista dei comandi",
+  "crafttweaker.command.description.info.block" : "Attiva o disattiva il lettore di blocchi. In modalità di analisi dei blocchi, facendo clic con il tasto destro su un blocco potrai visualizzare il nome, i metadati e i dati dell'entità-blocco di quel blocco.",
```

Comment:
Is "entità-blocco" the usual term used to refer to block entities?

Reply:
Yes, you can check the official translation at [the official Crowdin project](https://crowdin.com/project/minecraft).